modules/connector/applicationContext-connector.xml

```diff
@@ -26,6 +26,7 @@
 		<context:exclude-filter type="regex"
 			expression="org.opensrp.connector.rapidpro..*" />
     </context:component-scan>
+	<context:component-scan base-package="org.smartregister.stock.openlmis" />
```

Comment:
This should be optional, define this inside an openlmis profile

Reply:
done